lib/CSSStyleDeclaration-impl.js

```diff
@@ -48,7 +50,7 @@ CSSStyleDeclaration.prototype = {
    * @param {string} [priority=null] "important" or null
    * @see http://www.w3.org/TR/DOM-Level-2-Style/css.html#CSS-CSSStyleDeclaration-setProperty
    */
-  setProperty: function(name, value, priority) {
+  setProperty(name, value, priority) {
     if (value === undefined) {
```

Comment:
There are more things you can clean up here. For example, `value` can now only be a string.

Reply:
Actually, a good chunk of the implemented properties call `_setProperty` with `value = undefined`, expecting it to do nothing.

Reply:
Yeah, I understand. That's why I commented at this function in particular. I haven't checked other functions either fyi.